src/node/CarplayNode.ts

```diff
@@ -62,14 +86,24 @@ export default class CarplayWS extends EventEmitter {
   }
 
   start = async () => {
-    const { knownDevices } = DongleDriver
+    // Find device to "reset" first
+    let device = await this.findDevice()
+    await device.open()
+    await device.reset()
+    await device.close()
+    // Resetting the device causes an unplug event in node-usb
```

Comment:
Is this always needed? If we plug in a device and start this will we need to open, reset,close and then open again?

Reply:
Two questions in there, first, is the reset always needed? Not always, but better to do it just to be safe, sometimes the device is already half initialized or whatever so this means you don't have to replug it. Secondly, is the wait and closing always needed after a reset? Absolutely. That's what all the libusb errors that people are getting are from. A bunch of debugging and this is the solution I've arrived at

Reply:
Thanks for explaining!